tests/lib/pkgdb.sh

```diff
@@ -31,13 +31,24 @@ fedora_name_package() {
     esac
 }
 
+opensuse_name_package() {
```

Comment:
Why are they backwards` It's <distro>_<verb>_<what> were distro=opensuse verb=to name and what=package in this case.